src/pages/forms/login.js

```diff
@@ -0,0 +1,192 @@
+import React from "react";
+import { useState } from "react";
+import styles from "./login.module.css";
+import { Link } from "react-router-dom";
+import baseUrl from "../../url/environment";
+
+const Login = () => {
+  const initialValue = {
+    email: "",
+    password: "",
+  };
+
+  const [formValues, setFormValues] = useState(initialValue);
+  const [showPass, setShowPass] = useState();
+  const [formErrors, setFormErrors] = useState({});
+
+  const onChangeHandler = (e) => {
+    const { name, value } = e.target;
+    setFormValues({ ...formValues, [name]: value });
+  };
+
+  const onChangePassHandler1 = () => {
+    setShowPass(false);
+  };
+  const onChangePassHandler2 = () => {
+    setShowPass(true);
+  };
+
+  const onSubmitHandler = async (e) => {
+    e.preventDefault();
+    setFormErrors(validate(formValues));
+
+    if (errors.emailIndicator === false && errors.passwordIndicator === false) {
+      const response = await fetch(`${baseUrl}/v1/auth/email/login`, {
+        method: "POST",
+        body: JSON.stringify(formValues),
+        headers: {
+          "Content-Type": "application/json",
+        },
+      });
+      console.log(response);
```

Comment:
handle response here , like in success it should go into the same 
and in case of error , it show show the error to the user .